src/components/HeaderBtn/HeaderBtn.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+interface HeaderBtnProps extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
+  buttonType: 'icon-bell' | 'icon-mail';
+}
+
+export const HeaderBtn: React.FC<HeaderBtnProps> = ({ buttonType }) => {
+  return (
+    // eslint-disable-next-line jsx-a11y/anchor-is-valid
+    <a href="#" className="btn btn-sm btn-header">
```

Comment:
`a` tag is bad solution for the SPA application. needs to use Router link instead